corehq/util/test_utils.py

```diff
@@ -165,3 +168,30 @@ def inner(*args, **kwargs):
         return helper(*args, **kwargs)
 
     return inner
+
+
+class log_sql_output(ContextDecorator):
```

Comment:
@dimagi/team-commcare-hq FYI you can use this on a function or as a context manager to log sql output without having to do it for everything in the code.